articles/sentinel/azure-sentinel-billing.md

```diff
@@ -116,6 +116,7 @@ Manage data ingestion and retention:
 - [Optimize Log Analytics costs with dedicated clusters](#optimize-log-analytics-costs-with-dedicated-clusters).
 - [Separate non-security data in a different workspace](#separate-non-security-data-in-a-different-workspace).
 - [Reduce long-term data retention costs with Azure Data Explorer (ADX)](#reduce-long-term-data-retention-costs-with-adx).
+- Use Data Collection Rules for your Windows Security Events.
```

Comment:
@nayef-yassin add an xref like this:

- [Use data collection rules for your Windows security events](#use-data-collection-rules-for-your-windows-security-events)

Reply:
Really sorry as I am not exactly sure I did this properly - I copied the way it is  done in the bullets above...please do share the way to do it if you can.